experimental/PropertyDDS/packages/property-properties/src/test/properties/stringProperty.spec.js

```diff
@@ -176,6 +176,23 @@ describe('StringProperty', function () {
             expect(testString.getValue()).to.equal('ABBAAAA');
         });
 
+        it('Should report dirtiness correctly when introducing a modification in certain order', function () {
+            testString = PropertyFactory.create('String');
+            let newValue = "test";
+            const node = PropertyFactory.create('NodeProperty');
+            node.insert('stringProp', testString);
+            // Ignore insert changeset
+            node.cleanDirty();
+            node.applyChangeSet(JSON.parse(`{"modify":{"String":{"stringProp":{"value":"${newValue}","oldValue": ""}}}}`));
+            node.cleanDirty(BaseProperty.MODIFIED_STATE_FLAGS.PENDING_CHANGE);
+            node.cleanDirty();
```

Comment:
aren't these two redundant? Does cleanDirty without parameter clean all flags or only dirty?

Reply:
It should be all removed, but tried to simulate the exact order of function calls from inspector.